README.md

```diff
@@ -261,4 +261,37 @@ Delete the service resources by running the following ensuring your reference th
   terraform refresh -var-file=env/dev/dev.tfvars
 
   terraform destroy -var-file=env/dev/dev.tfvars
-  ```
\ No newline at end of file
+  ```
```

Comment:
Add more about why these linters have been used.

Reply:
Don't really think its necessary tbh, we have that info on KEH confluence page